server/v2/stf/branch/state.go

```diff
@@ -0,0 +1,81 @@
+package branch
+
+import (
+	"fmt"
+	"unsafe"
+
+	"cosmossdk.io/server/v2/core/store"
+)
+
+func NewWritersMap(
```

Comment:
There's some nomenclature confusion here. This says new `WritersMap` but returns `WriterMap`. Why not just call the type here `WriterMap` instead of `WritersMap`?

Reply:
fixed